guides/common/modules/proc_salt-using-content-management.adoc

```diff
@@ -0,0 +1,23 @@
+[id="salt_guide_using_content_management_{context}"]
+= Using {Project}s Content Management
+
+This section provides best practice recommendations on how to provide managed hosts with the required Salt Minion client software using {Project}.
```

Comment:
It's a procedure though, not a recommendation list?

Reply:
Reworded.